config/environments/production.rb

```diff
@@ -3,15 +3,27 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
   #Action Mailer
-config.action_mailer.default_url_options = {host: 'fashionxt-crm-service.herokuapp.com', protocol: "https"}
+
+#   ActionMailer::Base.smtp_settings = {
+#   :user_name => 'apikey', # This is the string literal 'apikey', NOT the ID of your API key
+#   :password => 'Rails.application.credentials.sendgrid_api_key', # This is the secret sendgrid API key which was issued during API key creation
+#   :domain => 'yourdomain.com',
+#   :address => 'smtp.sendgrid.net',
+#   :port => 587,
+#   :authentication => :plain,
+#   :enable_starttls_auto => true
+# }
+
+config.action_mailer.default_url_options = {host: 'fashionxt-crm-test.herokuapp.com', protocol: "https"}
```

Comment:
Nit: Break this into 2 lines.
```suggestion
config.action_mailer.default_url_options = {
  host: 'fashionxt-crm-test.herokuapp.com', 
  protocol: "https"
}
```

Reply:
I guess this will be a task for next Iteration, but we'll need to use environment variables to store the values.

Reply:
Yes we should use environment variables. I will make the changes for the next iteration.